Prace_domowe/Praca_domowa2/Piotr Marciniak/notebook.ipynb

```diff
@@ -0,0 +1,2120 @@
+{
```

Comment:
<p>Nie do końca o to chodziło. Chodziło o porównanie danych imputowanych z oryginalnymi, a Ty porównałeś wszystkie dane ze wszystkich kolumn (w tym oryginalne) z danymi oryginalnymi. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>